jannovar-filter/src/main/java/de/charite/compbio/jannovar/filter/GeneWiseInheritanceFilter.java

```diff
@@ -72,6 +72,7 @@ public GeneWiseInheritanceFilter(Pedigree pedigree, JannovarData jannovarDB,
 		List<String> contigs = new ArrayList<String>();
 		for (Chromosome chr : jannovarDB.getChromosomes().values()) {
 			contigs.add(chr.getChromosomeName());
+			contigs.add("chr"+chr.getChromosomeName());
```

Comment:
I would recommend using `JannovarData`'s dictionary as this also includes correct mappings for the mitochondrial chromosome, unplaced etc.